tensorboard/components/vz_line_chart/vz-line-chart.ts

```diff
@@ -858,6 +906,9 @@ class LineChart {
     if (this.smoothingEnabled) {
       this.smoothLinePlot.datasets(this.datasets);
     }
+    if (this.marginLinePlots) {
```

Comment:
If `this.marginLinePlots` is always a possibly-empty list (see comment above), then this guard is not required.